src/main/scala/diplomaticobjectmodel/model/OMRTLModule.scala

```diff
@@ -2,6 +2,47 @@
 
 package freechips.rocketchip.diplomaticobjectmodel.model
 
+// Examples of ClockRelationships
+//  {
+//   "clock0": "core_clock_0",
+//    "clock1": "clock",
+//    "relationship": "$clock0 == m * $clock1 where m is a positive integer",
+//  }
+
+//  {
+//    "clock1": "rtc_toggle",
+//    "relationship": "$clock0 >= 2 * $clock1",
+//  }
```

Comment:
Shouldn't this comment go next to the definition of `OMClockRelationship`? It's confusing to see this not next to the thing that it's actually related to.